lib/bamboo/adapters/sendgrid_helper.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Bamboo.SendgridHelper do
+  @moduledoc """
+  Functions for using features specific to Sendgrid template substitution tags.
+
+  Only one template can be specified, but multiple tags can be added to the
```

Comment:
This seems like something that might need a header. Like `## Using templates and tags together`

Also, should it be "tags" or should it be "substitutions"?